### DISCUSSION TITLE
Is it possible to cheat a pool keeping an amount for yourself?

### DISCUSSION THREAD

**Discussion by todordev** [Chia-Network/chia-blockchain#8535] [General]
Sometimes, the system store this information in the logs.

```
pool.collect_pool_rewards_loop():369 - Not claimable amount: 10.50
pool.collect_pool_rewards_loop():420 - Not buried amounts: 1.75
```

After few minutes the system stores the following information.

```
Wrong target puzzle hash: -----------
Singleton is not assigned to this pool

pool.collect_pool_rewards_loop():369 - Not claimable amount: 12.25
pool.collect_pool_rewards_loop():420 - Not buried amounts: 0.00
```

What does it mean?

People say, farmers leave the pool as soon as they win a block, to not share rewards.
Is it possible farmers to keep whole amount if they leave before an amount be claimed by the pool?

**Comment by loppefaaret** [comment]
are you running the reference pool code ? it is not meant to be run as such, it is a reference for you to build you own pool code - protect your claims, by ensuring that your pool claims rewards faster, than the farmers can leave the pool!

**Reply by todordev** [reply]
Yeah, we are using reference pool. 
I see. Thank you very much!

**Comment by BrandtH22** [comment]
@ITPrism , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
